tests/gui/components/update-notifier.spec.js

```diff
@@ -4,10 +4,19 @@ const m = require('mochainon');
 const angular = require('angular');
 const units = require('../../../lib/shared/units');
 const settings = require('../../../lib/gui/models/settings');
+const analytics = require('../../../lib/shared/analytics');
 require('angular-mocks');
 
 describe('Browser: UpdateNotifier', function() {
 
+  before(function() {
+    analytics.disableConsoleOutput();
+  });
+
+  after(function() {
+    analytics.enableConsoleOutput();
```

Comment:
I believe test frameworks set some global variables so that packages can know if they are running on a test suite. We should check what variables mocha expose and disable this altogether on corvus itself.